cfpb/hmda-frontend
# Remove UTF-8 encoding check from Filing and FFVT

Closes #1527 

- Removes client-side file encoding check from both Filing app and FFVT tool
- Applies the same client-side file validations that we run in the Filing app to the FFVT tool
- Adds language about verifying UTF-8 encoding in the message for missing transmittal sheet, since non-UTF-8 files seem to have additional characters ahead of the `1|` in LAR files
- Removes `detect-file-encoding-and-language` dependency

## Testing
- Run against 1GB UTF-16 encoded LAR file and the check completed almost instantly
- Run against properly formatted, UTF-8 encoded LAR file and no errors were triggered

## Screenshots

### Filing
<img width="1012" alt="Screen Shot 2022-09-09 at 11 43 19 AM" src="https://user-images.githubusercontent.com/2592907/189412397-ea919a44-0072-4694-be1f-dab18287e3d3.png">

### FFVT
<img width="918" alt="Screen Shot 2022-09-09 at 11 41 52 AM" src="https://user-images.githubusercontent.com/2592907/189412416-dfbadccc-ecfe-4422-b672-a70499ebb2be.png">
